app/Http/Requests/WordRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class WordRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'literal' => 'required|alpha_num|max:255',
```

Comment:
Den længst engelsk ord er på 189.819 anslag 😁Kilde: https://en.wikipedia.org/wiki/Longest_word_in_English

Reply:
Min pointe er at du skal passe på at begrænse antal anslag i en variable du ikke kender længden på (Især da det er brugerstyret)

Reply:
1) Prøv at læse 'meaning' for de to førstenævnte ord.
2) Layout vil jo fucke op, hvis ordet på 190000 bogstave skal vises. Vi skal sikre os at layout kan håndtere den værste case.

Reply:
Enige at layoutet ville fucke op... OG vores database... det er jo literal der har en begræsning på 255 char alligevel. Så bare behold det. Min research ser at den længest COMMON ord i verden ligger i længden 65-ish. Så det er fint nok.